website/docs/gettingstarted/installation.mdx

```diff
@@ -16,7 +16,7 @@ sidebar_position: 1
 Wails has a number of common dependencies that are required before installation:
 
 - Go 1.17+
-- NPM (Node 14+)
+- NPM (Node 15+)
```

Comment:
Why the node update?

Reply:
Because both the [base template](https://github.com/wailsapp/wails/blob/513c1fd981dc999f0e726b37376ca4ed28c420ed/v2/cmd/wails/internal/commands/generate/template/base/package.json#L13-L15) and [Vue template](https://github.com/misitebao/wails-template-vue/blob/ec24d9281cfe740b858c32d8f875d795e197884e/frontend/package.tmpl.json#L13) you add need the support of the npm workspace function, and this function requires `npm>=7.x`, since npm is released with Node, the corresponding Node version is `15.x.x`

This move is to avoid new users and users who are not familiar with front-end construction tools from wasting time due to additional environment problems during development, and the currently recommended version of Nodejs has reached `16.14.0`